scripts/export.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+artifacts_dir="build/contracts"
```

Comment:
nb: all these assume `pwd` is the root of the project.
if i run them from the `./scripts` directory, there are errors.

I think you used
`SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`
in many iov-core scripts, and may be quite useful to define eg.
`artifacts_dir="${SCRIPT_DIR}/../build/contracts"`